concepts/classes/about.md

```diff
@@ -0,0 +1,230 @@
+# classes
+
+Classes are a centric concept in [object-oriented programming][object-oriented-programming].
+[In Crystal so is everything an object][everything-is-an-object], and every object has a type, and it can respond to some methods.
+Classes are blueprints to create objects, providing initial values for state, like variables, and implementations of behavior, like methods.
+When creating an object from a class, the object is an instance of that class, the instance holds all the instance variables and states.
+
+## Class definition
+
+A class is defined using the `class` keyword, followed by the name of the class written in `UpperCamelCase`.
+
+```crystal
+class Account
+end
+```
+
+## Creating an instance
+
+To [create an instance of a class][new-initialize], you call the class name with the `new` method.
+
+```crystal
+class Account
+end
+
+account = Account.new
+```
+
+In the example above so is the variable `account` is an instance of the class `Account`.
+
+When creating an instance, there is a special method called `initialize` that is a constructor, it is called when an instance is created.
+The constructor allows you to set the initial state of the instance.
+The `initialize` method can take arguments, which are passed when creating an instance.
+
+```crystal
+class Account
+  def initialize(number)
+    p 5 + number
```

Comment:
What is p?

Reply:
In Ruby and swift they like short print statements so they use p, I think I explained that in an earlier concept.

Reply:
This "they use `p`" in Ruby is not true for the purpose of  "short print statements".  Reach out to me if you would like more of an explanation. But `p` is a debugging tool, not a shortcut for `puts` or `print` in Ruby.

Reply:
Short I mean that the method names use few letters not that they are a shortcut for another method.

"Ruby and Crystal they like short print statements so they use p"

What I meant in the line above is that they could equally have named it: inspect_print (or something longer than just p), but as far as I guess they chose p since it was short.

 p as you said is meant for debugging since it shows more "raw" data.